### PR TITLE
Update customer_account.xml

### PR DESCRIPTION
**Pull request by mageho**
Using block Magento\Customer\Block\Account\SortLinkInterface instead of Magento\Framework\View\Element\Html\Link\Current
to be able to sort link by sortOrder function

### CODE REVIEW

File: view/frontend/layout/customer_account.xml [hunk 1]
```diff
@@ -8,10 +8,11 @@
 <page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
     <body>
         <referenceBlock name="customer_account_navigation">
-            <block class="Magento\Framework\View\Element\Html\Link\Current" name="customer-account-navigation-privacy-link" after="customer-account-navigation-wish-list-link" ifconfig="gdpr/general/enabled">
+            <block class="Magento\Customer\Block\Account\SortLinkInterface" name="customer-account-navigation-account-privacy-link">
```
Comment (thomas-kl1) [inline view/frontend/layout/customer_account.xml:11]: Please keep `ifconfig="gdpr/general/enabled"` so the link can be enabled from the settings.

File: view/frontend/layout/customer_account.xml [hunk 2]
```diff
@@ -8,10 +8,11 @@
...
                 <arguments>
                     <argument name="path" xsi:type="string">customer/privacy/settings</argument>
-                    <argument name="label" xsi:type="string" translate="true">Privacy Settings</argument>
+                    <argument name="label" xsi:type="string" translate="true"><![CDATA[Privacy Settings]]></argument>
```
Comment (thomas-kl1) [inline view/frontend/layout/customer_account.xml:14]: Why do we need the CDATA markup here?

**Review by thomas-kl1 — CHANGES_REQUESTED** [review]

**Comment by thomas-kl1** [comment]
Hi @mageho ! Thank you a lot for your contribution! You are right we should use sortable element in the account menu.
Could you check my review?

**Comment by mageho** [comment]
Good morning Thomas,

You are welcome!

Here you go for your review

Ilan

**Review by thomas-kl1 — APPROVED** [review]

**Pull request merged**